Lib/dataclasses.py

```diff
@@ -1222,11 +1222,6 @@ def _get_slots(cls):
 
 
 def _update_func_cell_for__class__(f, oldcls, newcls):
-    # Returns True if we update a cell, else False.
```

Comment:
I think this is still the case, so we should keep this comment (or turn it into a docstring.)